boot/bootdata/hivecls.inf

```diff
@@ -241,15 +249,15 @@ HKCR,".ico","PerceivedType",0x00000000,"image"
 HKCR,"icofile","",0x00000000,"Icon"
 HKCR,"icofile","FriendlyTypeName",0x00020000,"@%SystemRoot%\system32\shell32.dll,-213"
 HKCR,"icofile\DefaultIcon","",0x00000000,"%1"
-HKCR,"icofile\shell\open\command","",0x00020000,"rundll32.exe %SystemRoot%\system32\shimgvw.dll,ImageView_Fullscreen %1"
+HKCR,"icofile\shell\open\command","",0x00020000,"rundll32.exe %SystemRoot%\system32\shimgvw.dll,ImageView_Fullscreen ""%1"""
```

Comment:
Does it work better with the quotes?

Reply:
In Windows 10: 
```
HKEY_CLASSES_ROOT\Applications\photoviewer.dll\shell\print\command=%SystemRoot%\System32\rundll32.exe "%ProgramFiles%\Windows Photo Viewer\PhotoViewer.dll", ImageView_Fullscreen %1
```
So, Windows doesn't needs double quotes for %1.

I think Windows does workarround on interpretation about %1.

Reply:
In Windows, mspaint.exe doesn't do this workarround.

Reply:
base/system/rundll32/rundll32.c (383):
```c
    // The rest of the arguments will be passed to dll function
    if (i < argc)
        lptCmdLine = argv[i];
    else
        lptCmdLine = _T("");
```

rundll32 passes the rest commandline, so it doesn't require to add quotes.

Reply:
https://support.microsoft.com/en-us/help/164787/info-windows-rundll-and-rundll32-interface

> Note in particular that this means a DLL in the "C:\Program Files" folder should be converted to its short name.
> The dllname may not contain any spaces or commas or quotation marks. This is a limitation in the Rundll command line parser.

Reply:
The problem is whether adding quotes affects 16-bit compatibility.

Reply:
A Win16 app can access the registry tree. He doesn't assume it was quoted.

Reply:
Also, as indicated in the URL you provided, everything after the name of the function in the rundll32 command line is passed to the function (here, "ImageView_Fullscreen"), so you should not touch (ie. add quotes) around it.

Reply:
Reverted.